website/content/en/highlights/2022-02-08-0-20-0-upgrade-guide.md

```diff
@@ -38,3 +39,21 @@ it.  For sets, however, we would simply clear their values and thus report a set
 behavior was not consistent with how we expired other metric types, and additionally, could still
 lead to a growth of unique series in a scrape, over time, as we would clear the set but not stop
 reporting it.
+
+#### New metrics behavior for `route` transform {#metrics-route-transform}
+
+Previously, a `route` transform was internally expanded into a separate
+transform for each defined output. Each of these expanded transforms then
+emitted its own metrics. Now, with improved internal support for components with
+multiple outputs, this expansion mechanism has been removed.
+
+A single `route` transform will now emit one set of metrics. When relevant,
+specific output information will be recorded as a metric tag `output`. For
+example, for a `route` transform `foo` with an output `first` configured, the
+`events_discarded_total` metric will now look like the following:
```

Comment:
Could we list all affected metrics here? That should make it easy for users to recognize and update their metrics queries.

Reply:
By all affected metrics (since technically all `route` metrics are affected), do you mean those we'd like to add the `output` tag to? Can do -- those should be `components_sent_*`, `events_discarded_total`.

Reply:
Yeah, exactly. I think busy users will appreciate having it spelled out for them rather than have them go look and see which metrics that could mean 😄

Reply:
Updated in 5fefe26